packages/core/src/indexing-store/cache.ts

```diff
@@ -526,67 +528,79 @@ export const createIndexingCache = ({
             `;
           const truncateQuery = `TRUNCATE TABLE "${getTableName(table)}" CASCADE`;
 
-          const text = getCopyText(
-            table,
-            updateValues.map(({ row }) => row),
-          );
-
-          await new Promise(setImmediate);
-
           const endClock = startClock();
 
+          // @ts-ignore
+          await client.query(createTempTableQuery);
+          // @ts-ignore
+          await client.query("SAVEPOINT flush");
+
           try {
```

Comment:
Does it make sense for these to be outside the try ? Just notice it's different from before.

Reply:
yes